src/utils/errors.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from "./localize";
+
+export class NoWorkspaceError extends Error {
```

Comment:
Not sure if we should follow this exactly but this file was put directly in `src/` in the functions repo: https://github.com/microsoft/vscode-azurefunctions/blob/main/src/errors.ts